packages/http-client-java/generator/http-client-generator-test/package.json

```diff
@@ -7,8 +7,8 @@
     "format": "npm run -s prettier -- --write",
     "check-format": "npm run prettier -- --check",
     "prettier": "prettier --config ./.prettierrc.yaml **/*.tsp",
-    "testserver-run": "npx cadl-ranch serve ./node_modules/@azure-tools/cadl-ranch-specs/http --coverageFile ./cadl-ranch-coverage-java.json",
-    "testserver-start": "npx cadl-ranch server start ./node_modules/@azure-tools/cadl-ranch-specs/http --coverageFile ./cadl-ranch-coverage-java.json",
+    "testserver-run": "npx cadl-ranch serve ./node_modules/@azure-tools/cadl-ranch-specs/http --coverageFile ./cadl-ranch-coverage-java-standard.json",
```

Comment:
I'm not sure how these `npx` calls slipped in.   It's generally unsafe to call `npx` without using the `--no` option (previously called `--no-install`) in a CI process.  This will require that the package be preinstalled in `node_modules` before calling it.

Reply:
Thanks for remind. I will add `--no-install` to all of them.